exchanges/graphcache/benchmarks/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@urql/exchange-graphcache-tachometer-benchmark",
+  "version": "1.0.0",
+  "description": "Comprehensive Tachometer benchmarks for urql/graphcache",
+  "main": "index.js",
+  "scripts": {
+    "addTodo": "tachometer addTodo.html",
```

Comment:
if we call these `bench:currentName` then we can do `"bench": "npm-run-all bench:*"`

Reply:
@JoviDeCroock
Something like this??
 
```
    "bench": "npm-run-all bench:*",
    "bench:addTodo": "tachometer addTodo.html",
    "bench:update": "tachometer updateTodo.html"
```